app/src/main/java/grinnell/appdev/edu/lyles/fragments/BeerFragment.java

```diff
@@ -1,22 +1,117 @@
 package grinnell.appdev.edu.lyles.fragments;
 
+import android.os.AsyncTask;
 import android.os.Bundle;
-import android.support.annotation.Nullable;
-import android.support.v4.app.Fragment;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.ListView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
 
 import grinnell.appdev.edu.lyles.R;
 
-/**
- * Created by Mattori on 5/9/16.
- */
-public class BeerFragment extends Fragment {
+public class BeerFragment extends AppCompatActivity {
+
+    ListView list;
```

Comment:
Name variables properly and use the necessary access modifiers. eg. `ListView list;` should read as `private ListView mBeerListView;`